Airports/EGCB/SMR/Labels.txt

```diff
@@ -1,16 +1,16 @@
-"B4" N053.28.15.541 W002.23.38.540 smrWhite
-"B3" N053.28.15.656 W002.23.34.564 smrWhite
-"B2" N053.28.16.462 W002.23.23.451 smrWhite
-"B1" N053.28.16.807 W002.23.17.942 smrWhite
-
-"D1" N053.28.12.487 W002.23.36.929 smrWhite
-"C1" N053.28.10.731 W002.23.27.013 smrWhite
+;Holding Points
 "A1" N053.28.14.013 W002.23.04.449 smrWhite
 "A2" N053.28.15.740 W002.23.01.575 smrWhite
 "A3" N053.28.17.237 W002.23.03.395 smrWhite
 "A4" N053.28.21.297 W002.23.10.006 smrWhite
+"B1" N053.28.16.807 W002.23.17.942 smrWhite
+"B2" N053.28.16.462 W002.23.23.451 smrWhite
+"B3" N053.28.15.656 W002.23.34.564 smrWhite
+"B4" N053.28.15.541 W002.23.38.540 smrWhite
 "A5" N053.28.25.270 W002.23.14.462 smrWhite
```

Comment:
A5 before the "B"s

Reply:
To be fair to me, this was one of the ones that my software said was so bad that it I'd have to do it manually, and we've already covered the fact that I'm blind and don't know alphanumeric ordering 😉